bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuntimeRule.java

```diff
@@ -28,6 +33,7 @@
 public class RuntimeRule extends Rule {
 
     private Map<String, Module> moduleMap;
+    private RuleStatusInfo statusInfo;
```

Comment:
This looks fishy. Afair, we so far kept the status of the rules in the rule engine, having it mixed up with the rule meta-data might not be the best idea...

Reply:
Ok, if we consider the RuntimeRules an internal data structure for the rule engine, it should be fine.